errbot/main.py

```diff
@@ -60,8 +60,13 @@ def setup_bot(backend_name, logger, config, restore=None):
     if not path.exists(botplugins_dir):
         makedirs(botplugins_dir, mode=0o755)
 
+    plugin_indexes = getattr(config, 'BOT_PLUGIN_INDEXES', ('http://errbot.io/repos.json',))
```

Comment:
I think it might be better to host this under a subdomain, something like `repos.errbot.io/repos.json`.

This will ensure it doesn't conflict with the sphinx-built generation so we don't have to interweave the two together, and would allow us to do HTTPS (if we don't put it on github pages) which I think we should absolutely do here.